publication-model/src/main/java/no/unit/nva/model/associatedartifacts/file/File.java

```diff
@@ -48,24 +44,12 @@ public abstract class File implements JsonSerializable, AssociatedArtifact {
     public static final String RIGHTS_RETENTION_STRATEGY = "rightsRetentionStrategy";
     public static final String PUBLISHED_DATE_FIELD = "publishedDate";
     public static final String UPLOAD_DETAILS_FIELD = "uploadDetails";
-    public static final Map<String, URI> LICENSE_MAP = Map.of("CC BY",
-                                                              URI.create("https://creativecommons.org/licenses/by/4.0"),
-                                                              "CC BY-NC", URI.create(
-            "https://creativecommons.org/licenses/by-nc/4.0"), "CC BY-NC-ND", URI.create(
-            "https://creativecommons.org/licenses/by-nc-nd/4.0"), "CC BY-NC-SA", URI.create(
-            "https://creativecommons.org/licenses/by-nc-sa/4.0"), "CC BY-ND", URI.create(
-            "https://creativecommons.org/licenses/by-nd/4.0"), "CC BY-SA", URI.create(
-            "https://creativecommons.org/licenses/by-sa/4.0"), "CC0", URI.create(
-            "https://creativecommons.org/publicdomain/zero/1.0"), "RightsReserved",
-                                                              URI.create("http://rightsstatements.org/vocab/InC/1.0/"));
```

Comment:
I'm tempted to say that there is a validation in the DTO, but we should define what is an acceptable license and (not least) how to inform the code of the list of potential URIs (per institution? How does that work for a shared record?) This probably requires a task in Jira

Reply:
An no, we shouldn't re-write URIs…just reject the bad ones